### PR TITLE
Add secrets for FPLA IDAM credentails

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1050

### Change description ###
Added publiclaw username and password secrets, which will be used to create Exception Record (and other CCD operations) for FPLA.

Flux PR - https://github.com/hmcts/cnp-flux-config/pull/2169

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/bulk-scan-orchestrator/values.yaml [hunk 1]
```diff
@@ -31,6 +31,8 @@ java:
         - idam-users-div-password
         - idam-users-probate-username
         - idam-users-probate-password
+        - idam-users-publiclaw-username
+        - idam-users-publiclaw-password
```
Comment (doncem) [inline charts/bulk-scan-orchestrator/values.yaml]: those only exist in aat. will service be ok to launch without secrets being present in other environments?

Comment (nitinprabhu) [reply charts/bulk-scan-orchestrator/values.yaml]: prod pod will fail if secrets are not there so we should add them I think in all environments.
But I agree we should just enable them in AAT https://github.com/hmcts/bulk-scan-orchestrator/blob/master/infrastructure/aat.tfvars#L4

Comment (nitinprabhu) [reply charts/bulk-scan-orchestrator/values.yaml]: ok this is now deprecated so secrets has to exist in prod.

Comment (aliveni) [reply charts/bulk-scan-orchestrator/values.yaml]: I added supported jurisdictions config to be able to disable publiclaw in prod. But, we still need to add idam credentials to prod key vault

Comment (aliveni) [reply charts/bulk-scan-orchestrator/values.yaml]: Updated  prod config to exclude `publilaw` https://github.com/hmcts/cnp-flux-config/pull/2169/files#diff-a81b4fc624fb7995c6f17e36a59a0540R28

Comment (aliveni) [reply charts/bulk-scan-orchestrator/values.yaml]: The idam secrets are now added to prod vault

Comment (doncem) [reply charts/bulk-scan-orchestrator/values.yaml]: i presume perftest can be done together with the ticket responsible deploying orchestrator to perftest

**Pull request merged**